src/components/QuranReader/index.tsx

```diff
@@ -66,6 +67,13 @@ const QuranReader = ({
         </div>
       </div>
       <Notes />
+      {lang === 'en' &&
+        quranReaderDataType === QuranReaderDataType.Chapter &&
```

Comment:
move this to `src/components/QuranReader/EndOfScrollingControls/ChapterControls.tsx` component.